test/e2e/allocator_test.go

```diff
@@ -183,13 +181,11 @@ func TestAllocatorCrossNamespace(t *testing.T) {
 			logrus.WithError(err).Info("failing Allocate request")
```

Comment:
Line 173 above 
```
   logrus.WithError(err).Info("failing http request")
```
Should be `"failing grpc.Dial"` as in other places.

Reply:
No this is actually failing the allocate request. grpc.Dial is successful at this point.

Reply:
I mean not in this exact place but above:
https://github.com/googleforgames/agones/blob/6a02b31331501ae5ed9bf844107af47ca254e8c4/test/e2e/allocator_test.go#L175

Reply:
Done.